pkg/mock/fileUtils.go

```diff
@@ -366,3 +366,7 @@ func (f *FilesMock) Chmod(path string, mode os.FileMode) error {
 
 	return nil
 }
+
+func (f *FilesMock) Abs(path string) (string, error) {
+	return "/root_folder/workspace/" + path, nil
```

Comment:
For self-consistency, you should re-use the existing toAbsPath private method. It will also take into account the current OS separator.

Reply:
It that really a good idea to use the OS dependent separator in the tests as that would mean that also my assertions need to be flexible / computed based in the OS ...

Reply:
The mock handles some of that. You can just AddFile and AddDir with forward slashes. You can also create the FileMock instance with a "Separator" parameter. Although that is not well tested. However, if the step-implementation is well written, then it will use filepath.Join() anyway, so your assertions may need to take that into account. It depends.

Reply:
In a TGIF project, someone (me?) could add a Join() method to both the real and mock FS, and replace all occurrences of filepath.Join() in the steps with that. Then test could look much less cluttered.

Reply:
done